style.css

```diff
@@ -140,16 +201,41 @@ a:hover {
     .gallery-column {
         flex-basis: 90%;
     }
+
 }
 
 @media screen and (max-width:400px) {
 
-
     .bottom-navbar_item--active {
         border: 0;
     }
 
     .bottom-navbar {
         font-size: 14px;
     }
+
+    .hero_item--desktop-only {
+        display: none;
+    }
```

Comment:
I think you should hide those sooner, so this doesn't happen:

<img width="565" alt="image" src="https://user-images.githubusercontent.com/11327761/204133372-f7177e11-6de3-47b9-bf62-c8a2defed51b.png">